pika/adapters/select_connection.py

```diff
@@ -189,24 +162,27 @@ class SelectPoller(object):
     to override the update_handler and start methods for additional types.
 
     """
-    TIMEOUT = 1
+    TIMEOUT = 5
+    # if the poller used MS specify 1000
+    TIMEOUT_MULT = 1
 
-    def __init__(self, fileno, handler, events, state_manager):
+    def __init__(self, state_manager):
         """Create an instance of the SelectPoller
 
-        :param int fileno: The file descriptor to check events for
-        :param method handler: What is called when an event happens
-        :param int events: The events to look for
         :param method state_manager: The method to manage state
 
         """
-        self.fileno = fileno
-        self.events = events
-        self.open = True
-        self._handler = handler
-        self._timeouts = []
+        self.filenos = dict()
```

Comment:
@wjps, I found the name `self.filenos` to detract from code readability. Perhaps something like `self.handlers` or `self.fileno_to_handler_map` would make it easier to follow the logic in the code.